config.js

```diff
@@ -503,4 +519,4 @@ config.WORM_ENABLED = false;
 config.ALLOW_BUCKET_CREATE_ON_INTERNAL = true;
 
 // Temporary flag to turn PostgreSQL as DB
-config.USE_POSTGRESQL = false;
+config.USE_POSTGRESQL = false;
```

Comment:
Can you change your editor/formatter to avoid removing ending newlines?

Reply:
Sure.